.github/workflows/regression-report.yml

```diff
@@ -50,16 +61,41 @@ jobs:
             exit 1
           fi
 
+  checkout_and_cache:
+    runs-on: ubuntu-latest
```

Comment:
`needs: [ check_pr_status]`
so it should NOT run is PR is not good.

Reply:
done

Reply:
also, i guess this can be resolved.